identity/client/src/pages/groups/modals/DeleteModal.tsx

```diff
@@ -29,7 +29,7 @@ const DeleteModal: FC<UseEntityModalProps<Group>> = ({
 
   const handleSubmit = async () => {
     const { success } = await callDeleteGroup({
-      groupKey: group.groupKey,
+      groupId: group.groupId,
```

Comment:
Only `groupId` is used from the `group` object, so it would be cleaner to destructure it directly in the props like

```ts
const DeleteModal: FC<UseEntityModalProps<Group>> = ({
  entity: { groupId },
  ...
```

then this and other usage will become much shorter 

```ts
callDeleteGroup({ groupId });
```